pact_broker/README.md

```diff
@@ -0,0 +1,160 @@
+# Pact Broker
+
+The Pact Broker is an application for sharing for consumer driven contracts and verification results. It is optimised for use with "pacts" (contracts created by the [Pact][pact-docs] framework), but can be used for any type of contract that can be serialized to JSON.
+
+**Why do I need one?**
+
+Contract testing is an alternative approach to traditional integration testing that gives you tests that are quicker to execute, and that are more maintainable at scale. One down side of the approach is that the important information that would be available all in one place at the end of an integration test suite execution (that is, the version numbers of all the applications that were tested together, and whether the tests passed or failed) is now spread out across many different builds. The Pact Broker is a tool that brings all this information back together again, and allows you to deploy with safety.
+
+It:
+
+* allows you to release customer value quickly and confidently by [deploying your services independently][decouple] and avoiding the bottleneck of integration tests
+* solves the problem of how to share contracts and verification results between consumer and provider projects
+* tells you which versions of your applications can be deployed safely together
+* automatically versions your contracts
+* allows you to ensure backwards compatibility between multiple consumer and provider versions (eg. in a mobile or multi-tenant environment)
+* provides API documentation of your applications that is guaranteed to be up-to date
+* shows you real examples of how your services interact
+* allows you to visualise the relationships between your services
+
+Features:
+
+* A RESTful API for publishing and retrieving pacts.
+* An embedded API browser for navigating the API.
+* Autogenerated documentation for each pact.
+* Dynamically generated network diagrams so you can visualise your microservice network.
+* Displays provider verificaton results so you know if you can deploy safely.
+* Provides a "matrix" of compatible consumer and provider versions, so you know which versions can be safely deployed together.
+* Provides badges to display pact verification statuses in your READMEs.
+* Enables an application version to be tagged (ie. "prod", "feat/customer-preferences") to allow repository-like workflows.
+* Provides webhooks to trigger actions when pacts change eg. run provider build, notify a Slack channel.
+* View diffs between Pact versions so you can tell what expectations have changed.
+* [Docker Pact Broker][docker]
```

Comment:
Should we update the references from the DiUS pact-broker to the Pact Foundation pact-broker @mefellows @bethesque ?

Reply:
Yes, good idea